File: leetcode/easy/squares-of-a-sorted-array/squares-of-a-sorted-array.py
# Given an integer array nums sorted in non-decreasing order, return an array of the squares of each number sorted in non-decreasing order.

class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        sq_nums = []
        for num in nums:
            sq_nums.append(num ** 2)
        return sorted(sq_nums)

File: leetcode/easy/squares-of-a-sorted-array/squares-of-a-sorted-array.js
// Given an integer array nums sorted in non-decreasing order, return an array of the squares of each number sorted in non-decreasing order.

/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortedSquares = function(nums) {
    let sq_nums = []
    nums.map((num) => {
        sq_nums.push(num ** 2)
    })
    return sq_nums.sort(function(a, b) {return a - b})
};